src/scene/scene.cpp

```diff
@@ -440,6 +441,60 @@ SceneGraphNode* Scene::loadNode(const ghoul::Dictionary& dict) {
     return rawNodePointer;
 }
 
+void Scene::addInterpolation(properties::Property* prop, float durationSeconds) {
```

Comment:
A change for the future: I think properties shouldn't really live in the scene. (Since there are non-scene properties.) I think it would make more sense to have a `PropertyManager` or similar. With the current structure this makes sense though.